test/test.js

```diff
@@ -112,8 +112,8 @@ describe('loadParsedJson', () => {
 
   describe('with a path to a file that does not exist', () => {
 
-    it('should throw an error', () => {
-      loader.loadParsedJson.bind(null, '.', 'does-not-exist').should.throw(/ENOENT/);
+    it('should not throw an error', () => {
+      loader.loadParsedJson.bind(null, '.', 'does-not-exist').should.not.throw();
```

Comment:
Very true. It was less typing to not modify the test more than adding the `not` word :)